lib/rubocop/cop/rspec/describe_method.rb

```diff
@@ -17,16 +17,24 @@ module RSpec
       #   describe MyClass, '.my_class_method' do
       #   end
       class DescribeMethod < Base
-        include RuboCop::RSpec::TopLevelDescribe
+        include RuboCop::RSpec::TopLevelGroup
 
         MSG = 'The second argument to describe should be the method '\
               "being tested. '#instance' or '.class'."
 
-        def on_top_level_describe(_node, (_, second_arg))
-          return unless second_arg&.str_type?
-          return if second_arg.str_content.start_with?('#', '.')
+        def_node_matcher :second_argument, <<~PATTERN
+          (block
+            (send #{RSPEC} :describe _first_argument $(str _) ...) ...
+          )
+        PATTERN
 
-          add_offense(second_arg)
+        def on_top_level_group(node)
+          second_argument = second_argument(node)
```

Comment:
we implicitly check that this is a `describe` this way. I'm not strictly against it, but since some cops need to operate only on describe, I wonder if we can't customize the on_top_level_group to actually have separate callbacks for on_top_level_describe. We still use the same code, just call different method if it is available and the top level group is a describe. And we don't make any breaking change in the API

Reply:
Implemented separate hooks, `on_top_level_example_group` and `on_top_level_group` 👍

Reply:
We still check implicitly if this is a describe, by using the second_argument matcher